bigbluebutton/greenlight
# Preupload presentation Front-end (Preview existing presentation)

## Description

- Front end changes to display current uploaded presentation attached to room
- Displays preview of representable files (images, videos, and pdf) and a file icon for non representable files 
## Testing Steps

- Pull code changes
- Create room
- Upload presentation to room
- Make sure room is displayed correctly
## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/38328371/173673973-786724f0-49b8-43b6-9dc8-63b6640c4d48.png)

![image](https://user-images.githubusercontent.com/38328371/173674002-a98637ff-29e4-4b9d-be3c-6e6bc694d81d.png)
